COLLABORATOR_GUIDE.md

```diff
@@ -76,6 +76,24 @@ where a pull request:
 
 The CC Members should serve as the final arbiter where required.
 
+### Cross-coordination Guidelines to Other Governing Groups
+All Board requests should be communicated to the Individual Member Directors from the 
+CommComm Chairperson each month as resolutions or updates. Directors present
+at the Board session are responsible for championing these items.
+  
+Coordinate with the TSC Director and Chairperson to ensure all relevant CommComm items 
+have been familiarized ahead of the Board meeting. The Chairperson of the CommComm is 
+responsible for communicating to the TSC chair any cross-concerning discussions, 
+attending meetings to coordinate efforts, and see how we need get input from each group.
+  
+Agenda items need to be emailed to the Node.js Executive Director the Thursday prior to 
```

Comment:
Can we add a qualifier of who should email the ED for clarity's sake? e.g. "The Individual Member Directors need to email agenda items to the Node.js Executive Director"